pkg/netpol/eval/internal/k8s/policy_connections.go

```diff
@@ -94,19 +94,27 @@ func (pc *PolicyConnections) CollectConnsFromLowerPolicyType(otherConns *PolicyC
 	otherConns.AllowedConns.Subtract(pc.DeniedConns)
```

Comment:
no, this func will "convert" `pc.PassConn` to allowed or denied based on the otherConns.AllowedConns and otherConns.DeniedConns.

the idea from this func is that the current `pc` contains the collected final conns from all the `ANPs` 
which means: 
- `pc.AllowedConns` are definitely allowed
-` pc.DeniedConns` are definitely denied 

and pc.PassedConns will be determined by the new `otherConns` object which is coming from a PolicyType with lower priority than ANP (i.e. a NetworkPolicy or a BANP);
so after subtracting the finite allowed and denied conns of `pc` from `otherConns` (lines 94-94) ; otherConns will contain only new allowed and denied conns. 
so we won't subtract the `otherConns.AllowedConns` from the pc.PassConn;
neither subtracting `otherConns.DeniedConns` from `pc.PassConns` 
vice versa the `otherConns.AllowedConns` will be added to (union) the `pc.allowedConns`
and the `otherConns.DeniedConns` will be added to the `pc.DeniedConns` 
hopefully that by this all `pc.PassConns` are covered

Reply:
yes if the "passed" connection was not restricted nether by regular netpol, neither by BANP : then it is not covered and will be "converted" to be allowed as a system default allow (in `check.go`).

example : 
we have on the resorces:
ANP passes `TCP80` from `x->y` (policy_conns: allow: empty, deny: empty, pass: TCP80)
there is no NP capturing this connection   --- (delegate to banp)
there is a BANP that capturing egress  from x to y but only on UDP52 (allow) (policy-conns of banp: allow: UDP52, deny empty, pass: empty)

returned result from this fucn : (allow: UDP 52, deny : empty, pass: TCP80)
in the calling func in `check.go lines 423-425` we see that the pass is not empty and add it to the allowed, since what is not restricted is allowed by default